peer/pendingheap/config_test.go

```diff
@@ -23,17 +23,58 @@ package pendingheap
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"go.uber.org/yarpc/api/peer"
 	"go.uber.org/yarpc/peer/hostport"
 	"go.uber.org/yarpc/yarpcconfig"
 	"go.uber.org/yarpc/yarpctest"
 )
 
 func TestPendingHeapConfig(t *testing.T) {
+	minus1, zero, twenty := -1, 0, 20
```

Comment:
Good: use variable to avoid hard code. Maybe it's better to make them constant if we don't modify these variables.

Reply:
unfortunately can't take the address of a `const` 
eg: https://play.golang.org/p/Ozm3bHEx-tF